cdc/sink/ddlsink/mq/mq_ddl_sink.go

```diff
@@ -72,6 +72,8 @@ type DDLSink struct {
 	statistics *metrics.Statistics
 	// admin is used to query kafka cluster information.
 	admin kafka.ClusterAdminClient
+	// connRefresherForDDL is used to refresh the connection for DDL events.
+	connRefresherForDDL kafka.SyncProducer
```

Comment:
I think it is a redundant parameter, you can modify the `DDLProducer` interface to add `Heartbeat` method.

Reply:
The DDLProducer interface has multiple implementations. Adding a method like Heartbeat solely for this specific case, which is not needed by other implementations, is unnecessary and could cause confusion for users of those other implementations.